### PR TITLE
Issue #885 Making sure that the --enable flag works as part of 'minis…

### PR DESCRIPTION
**Pull request by hferentschik**
…hift addons install'

Addresses #885

### CODE REVIEW

**Comment by hferentschik** [comment]
FYI, the actual fix is fine. The added test works on my dev machine. I am trying to figure out why it does not work on all CI systems.

**Comment by hferentschik** [comment]
Test is fixed now.

**Comment by LalatenduMohanty** [comment]
PR looks fine to me. I need to test it and then merge it though.

File: cmd/minishift/cmd/addon/addon_install.go [hunk 1]
```diff
@@ -74,11 +74,13 @@ func runInstallAddon(cmd *cobra.Command, args []string) {
 	if err != nil {
 		atexit.ExitWithMessage(1, fmt.Sprintf(failedPluginInstallation, err.Error()))
 	}
+	fmt.Println(fmt.Sprintf("Addon '%s' installed", addOnName))
```
Comment (budhrg) [inline cmd/minishift/cmd/addon/addon_install.go]: Why not `fmt.Sprintf("Addon '%s' installed\n", addOnName)` ?

**Comment by budhrg** [comment]
LGTM :+1: 

```
$ minishift addons install addons/eap-templates/ --enable
Addon 'eap' installed
Addon 'eap' enabled

$ minishift addons list
- eap            : enabled    P(0)
```

**Comment by budhrg** [comment]
Merged and thanks!

**Pull request closed without merging**